src/components/KeyboardAwareScrollView/index.tsx

```diff
@@ -113,6 +113,12 @@ const KeyboardAwareScrollView = forwardRef<
 
     const { height } = useWindowDimensions();
 
+    const restStyle = (Array.isArray(rest.style) && rest.style?.[0]) || {};
+    const scaleStyle =
+      "transform" in restStyle &&
+      (restStyle?.transform?.[0] as { scale?: number });
+    const inverted = (scaleStyle && scaleStyle?.scale === -1) || false;
+
```

Comment:
Ideally I would create a function `isInverted` (should be a pure function), I think you can put it in `utils`

And I would add `useMemo` here to avoid expensive checks 🤔 

So the code should be:

```tsx
const inverted = useMemo(() => isInverted(rest.style), [rest.style]);
```

What do you think?

Reply:
@lpatrun just a friendly ping 👋